tests/format/js/bracket-spacing/jsfmt.spec.js

```diff
@@ -1,2 +1,10 @@
 run_spec(__dirname, ["babel", "flow", "typescript"]);
-run_spec(__dirname, ["babel", "flow", "typescript"], { bracketSpacing: false });
+run_spec(__dirname, ["babel", "flow", "typescript"], {
+  // [prettierx]: broken-out bracket spacing options
+  graphqlCurlySpacing: false,
+  exportCurlySpacing: false,
+  importCurlySpacing: false,
+  objectCurlySpacing: false,
+  typeCurlySpacing: false,
+  yamlBracketSpacing: false,
```

Comment:
Do we need *all* of these broken-out spacing options here?

Reply:
Whoops, forgot to whittle these down. Just pushed an update that keeps only necessary options.